command/agent/command.go

```diff
@@ -472,10 +472,20 @@ func SetupLoggers(ui cli.Ui, config *Config) (*logutils.LevelFilter, *gatedwrite
 
 	// Create a log writer, and wrap a logOutput around it
 	writers := []io.Writer{logFilter}
-
```

Comment:
I was double-checking this before merging and noticed that `config.LogLevel` isn't coerced to upper-case until it's consumed (see line 464 above). It's coerced to a hclog enum in a different function in the agent code, so canonicalizing the case at the point we initially handle it is going to have a bunch of ripple effects. So let's canonicalize it here for use in this function:

```suggestion
		logLevel = strings.ToUpper(config.LogLevel)
```

Reply:
I have made changes according to your suggestion. Thanks for pointing it out !